client/src/lib/api.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+import config from '../env/config.js';
+
+
+// This first section contains helper/template functions to keep our other functions
+// simple and DRY. Reuse them! But please don't edit them without team buy-in, because all
+// of our individual API functions rely on them.
+const api = {};
+
+const url = `https://app-hrsei-api.herokuapp.com/api/fec2/${config.CAMPUS_CODE}`;
+
+const get = (endpoint, params = {}) => {
+  return axios.get(`${url}/${endpoint}/`, {headers: {Authorization: config.AUTH}, params: params})
```

Comment:
I think we should consider using NPM paths here to join the url and endpoints instead to work around any problems we may run into hardcoding separations like "/".

Reply:
good idea! I'm open to that. I can work on that in v2.